### DISCUSSION TITLE
External Database support for Annotation list

### DISCUSSION THREAD

**Discussion by rs-11** [grafana/grafana#56600] [General]
I started to use to Annotation feature, and also add some time-periods as annotations to my influx database as supported by Grafana. Unfortunately, those Annotations stored in the influx-db are not shown in the Annotation List Panel. Would be great to find the desired time-periods faster!

Thank you!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
